cmd/incus/project.go

```diff
@@ -1184,7 +1184,10 @@ func (c *cmdProjectGetCurrent) Run(cmd *cobra.Command, args []string) error {
 	if !ok {
 		return fmt.Errorf(i18n.G("Remote %s doesn't exist"), conf.DefaultRemote)
 	}
-
-	fmt.Println(remote.Project)
+	if remote.Project == "" {
+		fmt.Println("default")
```

Comment:
Instead of passing the `"default"` string, it'd be preferable to use `api.ProjectDefaultName`.

Reply:
Thanks for the notice, added a commit fixing it the good way.